### ISSUE TITLE
Fixed check context for dynamic stylesheet

### ISSUE THREAD

**Issue by plazorax** [themeum/kirki#2103]
In the latest versions, kirki uses the following method to display dynamic styles in the editor:

`add_action( 'enqueue_block_editor_assets', array( $this, 'enqueue_styles' ), 100 );`

Which adds dynamic styles to <head>

`<link rel='stylesheet' id='kirki-styles-css'  href='https://example.com?action=kirki-styles' media='all' />`

If you register a field with context settings

```
Kirki::add_field(
	'csco_theme_mod', array(
		'type'     => 'color',
		'settings' => 'color_primary',
		'label'    => esc_html__( 'Primary Color', 'example' ),
		'section'  => 'design',
		'priority' => 10,
		'default'  => '#FFFFFF',
		'context'  => array( 'front' ),
		'output'   => array(
			array(
				'element'  => '.content a',
				'property' => 'color',
			),
		),
	)
);
```
they will also be used in WP Editor (which is not true).

This is because the check below does not work on the link - /?action=kirki-styles
https://github.com/aristath/kirki/blob/develop/modules/css/class-kirki-output.php#L246

I propose such a solution:
https://github.com/aristath/kirki/pull/2102

**Comment by sergeysedykh** [comment]
Hi, @aristath! Just wanted to confirm the issue.

It's quite critical as styles from the fields with the context parameter will be applied to the whole WordPress dashboard resulting in overrides of default WordPress buttons, inputs, etc.

Here's a screenshot:
![image](https://user-images.githubusercontent.com/36392769/53547395-fae80980-3b3f-11e9-8cbc-4e4fd7e19ae6.png)

Would highly appreciate if you could take a look at the issue.

**Comment by aristath** [comment]
Merged the pull-request from #2102, tweaked it a bit in https://github.com/aristath/kirki/commit/911150813353008371e8a4654f106cb100bc9bfc and released v3.0.38.

**Comment by sergeysedykh** [comment]
Thank you so much. You're awesome! Kirki is ❤️
